src/com/googlecode/jmxtrans/model/Query.java

```diff
@@ -78,6 +81,14 @@ public String getResultAlias() {
 	}
 
 	/**
+	 * The useObjDomainAsKey property allows you to specify the use of the Domain portion of the Object Name
+	 * as part of the output key instead of using the ClassName of the MBean which is the default behavior.
+	 */
+	public boolean isUseObjDomainAsKey() {
```

Comment:
I posit that naming things is one of the hardest parts of software engineering. I routinely get it wrong. :cry: